src/main/kotlin/no/nav/syfo/client/pdl/PdlClient.kt

```diff
@@ -15,11 +16,35 @@ import org.slf4j.LoggerFactory
 
 class PdlClient(
     private val azureAdClient: AzureAdClient,
+    private val redisStore: RedisStore,
     private val baseUrl: String,
     private val clientId: String,
 ) {
     private val httpClient = httpClientDefault()
 
+    suspend fun hasAdressebeskyttelse(
+        callId: String,
+        personIdent: PersonIdentNumber,
+    ): Boolean? {
```

Comment:
Er det viktig å vite at `person` kallet feiler? Eller kunne vi defaultet til `false` slik at returtypen her blir `Boolean`? 😊

Reply:
Jeg har litt lyst til å bevare funksjonaliteten i apiet slik det var opprinnelig dvs at vi bare returnerer skjermingskode for de personene som gir OK-respons fra PDL. Føles også skummelt å defaulte til at person ikke har adressebeskyttelse når vi ikke vet.

Reply:
Vi eksponerer også at vi ikke vet om personen har adressebeskyttelse altså? Da er jeg enig i at det gir mening å beholde 😊

Reply:
Ja, eller implisitt gjennom at man sender en liste med personidenter som body til requesten og så får man tilbake kun de hvor vi vet skjermingskode (og som veileder har tilgang til).